### ISSUE TITLE
adapt-quicknav not working with the contrib-vanilla theme

### ISSUE THREAD

**Issue by kang17** [cgkineo/adapt-quicknav#24]
Using the latest version AT, when adding it to a course using the contrib-vanilla theme, it breaks preview with the dreaded "error occurred : contact admin"

output is :

error: [05 May 2017 13:19:49 -04:00] Error: Command failed: grunt server-build:d
ev --outputdir=courses\58f65fdabd661c0023b6c327\590cb2afd3312fac1132997a\build -
-theme=adapt-contrib-vanilla --menu=adapt-contrib-boxMenu

error: [05 May 2017 13:19:49 -04:00] Error: Command failed: grunt server-build:d
ev --outputdir=courses\58f65fdabd661c0023b6c327\590cb2afd3312fac1132997a\build -
-theme=adapt-contrib-vanilla --menu=adapt-contrib-boxMenu

In the animated GIF demo on this page it looks like the Harsh theme was used.  adapt-quicknav seems to work with Harsh and other themes, but consistently breaks preview when switched back to adapt-contrib-vanilla theme. any ideas what's going on?

**Comment by moloko** [comment]
hi @kang17 we (and others) use this extension with the authoring tool and vanilla theme quite a lot and have not run into anything like this, it's more likely to be something going a bit wrong in the authoring tool.

certainly I can say I have just tested this extension with the vanilla theme in the framework and it works fine.

as for the animated GIF demo - that's something I made, I can't remember for sure whether I was using vanilla or the [kineo theme](https://github.com/cgkineo/adapt-kineo-theme) at the time (I would guess vanilla) but it definitely wasn't the Harsh theme.
